Jenkinsfile

```diff
@@ -6,8 +6,7 @@ elifePipeline {
         commit = elifeGitRevision()
     }
 
-    elifeOnNode(
-        {
+    node('containers-jenkins-plugin') {
             stage 'Build images', {
```

Comment:
Indentation could be reduced?

Reply:
Yes, I didn't want to pollute the diff too much

Reply:
That’s what `?w=1` is for. :)